v4/source/channels.yaml

```diff
@@ -628,12 +628,12 @@
         - channels
       summary: Delete a channel
       description: >
-        Soft deletes a channel, by marking the channel as deleted in the
-        database. Soft deleted channels will not be accessible in the user
-        interface. Direct and group message channels cannot be deleted.
+        Archives a channel. This will set the `deleteAt` to the current timestamp in the database. Soft deleted channels may not be accessible in the user
+        interface. They can be viewed and unarchived in the system console > channel based on your license. Direct and group message channels cannot be deleted.
```

Comment:
```suggestion
        interface. They can be viewed and unarchived in the **System Console > User Management > Channels** based on your license. Direct and group message channels cannot be deleted.
```

Reply:
I'm a bit unclear about the "based on your license" phrase in this description.

Reply:
To my knowledge, not all license levels are able to do channel management within the system console. With all the license changes, I'm not sure where this line is drawn anymore.

Reply:
Great point! Channel management is available in [Mattermost Enterprise](https://docs.mattermost.com/configure/configuration-settings.html#channels). I've updated my recommendation to account for this.